### ISSUE TITLE
Not working on D7200

### ISSUE THREAD

**Issue by puhitaku** [puhitaku/mtplvcap#2]
https://www.reddit.com/r/Nikon/comments/hzcue6/software_to_turn_your_nikon_cam_into_a_webcam_on/fzi9u7f/

**Comment by puhitaku** [comment]
Currently, mtplvcap lacks report of reason when it fails to open a device

**Comment by puhitaku** [comment]
Aaand MTP extension string may differ between DSLR generations:

```
[0000]  INFO usb: found: 04b0:0439
[0000] FATAL mtp: failed to detect MTP devices: could not open 04b0:0439: mtp: no MTP extensions in 'Microsoft.com/DeviceServices:'
```

It's fixed in 1.0.2. Now waiting for further report.

**Comment by puhitaku** [comment]
Now D7200 is verified
